equality.go

```diff
@@ -24,13 +27,33 @@ func EqualFloat32(reporter interface{}, want, got float32) {
 	}
 }
 
+// EqualFloat32Tol - asserts that two floats are the same,
+// allowing for (relative) tolerance given as a parameter
+func EqualFloat32Tol(reporter interface{}, want, got, relTol float32) {
+	if want == 0.0 && got != 0.0 {
+		reportError(reporter, &failedFloatCompMsg{float64(want), float64(got)})
+	} else if m.Abs(float64((want-got)/want)) > float64(relTol) {
+		reportError(reporter, &failedFloatCompMsg{float64(want), float64(got)})
```

Comment:
we could also write a function for abs() without importing whole math library:
func abs(x float64) float 64
{
  return x * ((-1)*(x<0)+ (x>=0))
}

Reply:
👍 that sounds like a good idea to me